### ISSUE TITLE
VWWHealthPermission

### ISSUE THREAD

**Issue by emonster** [zakkhoyt/VWWPermissionKit#6]
Do you have an estimate on when VWWHealthPermission maybe implemented? Thank you very much!

**Comment by zakkhoyt** [comment]
Thanks, I'm working on HealthKit support now. There are a few differences between HealthKit and the other permissions that I need to work around, but it should be doable. I should have a new build out in a week or two. 

I will let you know (reply here) when I publish it. 

-Zakk

**Comment by zakkhoyt** [comment]
If you're in immediate need, I believe I've got HealthKit working. Use this commit:

https://github.com/zakkhoyt/VWWPermissionKit/tree/9c1b3fa388453c96c64fd88c050441a8cf05bc84

Note that when you generate your VWWHealthPermission object it uses a different convenience initializer.  Don't forget to add HealthKit to your Capabilities. 

// Sample usage for HealthKit

```
NSSet *shareTypes = [NSSet setWithObject:[HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierBloodAlcoholContent]];
NSSet *readTypes = shareTypes;

NSArray *permissions =
@[
  [VWWHealthPermission permissionWithLabelText:@"healthKit" shareTypes:shareTypes readTypes:readTypes]
  ];

// Using requirePermissions:permissions, the user cannot proceed until all permissions are authorized
[VWWPermissionsManager requirePermissions:permissions
                                    title:@"We need your approvoal before we get running"
                       fromViewController:self
                             resultsBlock:^(NSArray *permissions) {
                                 [permissions enumerateObjectsUsingBlock:^(VWWPermission *permission, NSUInteger idx, BOOL *stop) {
                                     NSLog(@"%@ - %@", permission.type, [permission stringForStatus]);
                                 }];
                             }];
```

**Comment by emonster** [comment]
Thank you Zakk!! Very helpful utility :)
I'll give it a try and look for the final release

**Comment by zakkhoyt** [comment]
Great. Please let me know if you find an issue.

Enjoy.

**Comment by emonster** [comment]
Hi Zakk,
Yes I do have a couple of questions.
Is string localisation in the roadmap?
Also if the permissionWithLabelText: string is too long it gets truncated, Would it be possible to support multiline label text?
Thank you!

**Comment by zakkhoyt** [comment]
Localization I haven't really thought about too much until (coincodentally)
a couple of hours ago. I've never really added support for localization to
any project, but I am educated on how to do it. I'll put it in the issues
list but it'll probably be lower priority.

Label, yes! I'm working appearance customization. You'll be able to pass
tintColor, authorizedColor, deniedColor, etc... You'll be able to specify
an image for each permission too. I'll be sure to support longer labels.
This should be soon.

-Zakk

**Comment by zakkhoyt** [comment]
If you're proficient at localizaion, feel free to send a pull request.
